packages/event_store/src/models/genesisAccount.js

```diff
@@ -0,0 +1,37 @@
+const { Model } = require('sequelize');
+
+module.exports = (sequelize, DataTypes) => {
+    class GenesisAccount extends Model {
+        static TYPE_VALIDATOR = 'validator';
+        static TYPE_DELEGATOR = 'delegator';
+        static TYPE_ACCOUNT = 'account';
```

Comment:
I assume these types are for their genesis states only, as post-genesis, an account can transition between any of these types, and even hold multiple types at once?

Reply:
Yes, those are mostly for the documentation purposes. The genesis accounts table never changes after it's populated with records.